eip-0027.md

```diff
@@ -0,0 +1,256 @@
+
+* If the block reward **R >= 15 ERG**, send **12 ERG** to the re-emission contract.
+* otherwise, if **R < 15 ERG**, send **R - 3 ERG** to the re-emission contract
+
+After the end of the current emission (starting from block #2,080,800), the re-emission contract works like the existing emission contract but does not have the emission curve encoded, only a flat payout.
+
+* Pay **3 ERG each block** from the re-emission contract.
+
+
+Updated Emission Details
+------------------------
+
+With the updated emission schedule described above, re-emission (with 3 ERG re-emission rewards per block) would be enough for 4,566,336 blocks (approx. 17.38 years).
+
+General Design
+--------------
+
+Ergo's emissions are controlled via a smart contract which has existed since before the genesis block (pre-genesis state). 
+
+This EIP proposes using the existing contract in combination with checks performed in the core protocol, which are only mandatory for mining nodes. Non-updated nodes will continue to successfully validate all transactions valid for the new nodes (both checking and not checking the new rules) after activation, making this change a **soft-fork**. 
+
+This EIP offers the following procedure for achieving this:
+
+* Inject two new tokens into the emission contract. 
+  * The first token is a *singleton* issued to mark the emission box (It could also be used to track the emission box efficiently). 
+  * The second token is the re-emission token, which will go into mining reward boxes. 
+ 
+* The amount of re-emission tokens in a mining rewards box shows how many ERG a miner should send to the re-emission contract when spending the box. Any miner can do the injection, which happens on activation height.
+
+We introduce two new contracts which are described in greater detail below. 
+* The **re-emission contract** pays 3 ERG for each block after the end of emission.
+
+* The **pay-to-reemission** contract, is a proxy contract active before the re-emission contract. 
+
+Miners are enforced to pay to this proxy contract. The contract is similar to the existing fee contract but with no time lock. 
+
+* We create new consensus-level checks that verify that the proper amount of re-emission tokens are going from the emission box to a miner rewards box, and also that during spending a box which is not an emission box but contains re-emission tokens (miner rewards box), the tokens are being burnt, and the same amount of nanoERG is locked by the re-emission contract. 
+
+The checks can be switched off via a soft-fork (via voting on disabling rule #123).
+
+Contracts
+-------------------
+
+**Re-emission contract**: This contract pays 3 ERG per block starting from block #2,080,800 . Also, this contract allows for merging with other boxes; the merging transaction must have only two outputs; the first output is for the re-emission contract, the second one is to pay a mining fee (its value can be 0.01 ERG at most)
```

Comment:
Please describe merging part in more details. Why it is called `sponsored` in the contract?

Reply:
Renamed to `merging`, what should be added into description?

Reply:
Renamed to `merging`, what should be added into description?